ios/GaloyApp/Info.plist

```diff
@@ -101,64 +101,10 @@
 	<string>Import a screenshot with a QR code</string>
 	<key>UIAppFonts</key>
 	<array>
-		<string>Roboto-Bold.ttf</string>
-		<string>Roboto-Medium.ttf</string>
-		<string>Roboto-Black.ttf</string>
-		<string>AntDesign.ttf</string>
-		<string>Entypo.ttf</string>
-		<string>EvilIcons.ttf</string>
-		<string>Feather.ttf</string>
-		<string>FontAwesome.ttf</string>
-		<string>FontAwesome5_Brands.ttf</string>
-		<string>FontAwesome5_Regular.ttf</string>
-		<string>FontAwesome5_Solid.ttf</string>
-		<string>Foundation.ttf</string>
 		<string>Ionicons.ttf</string>
-		<string>MaterialIcons.ttf</string>
-		<string>MaterialCommunityIcons.ttf</string>
-		<string>SimpleLineIcons.ttf</string>
-		<string>Octicons.ttf</string>
-		<string>Zocial.ttf</string>
-		<string>Fontisto.ttf</string>
```

Comment:
Looks like we removed all the font's aside from `Ionicons`.  Was there a reason we done this?

Reply:
reducing build size

Reply:
react native elements uses [react-native-vector-icons](https://reactnativeelements.com/docs/components/icon) library for components.  It requires these native fonts for rendering the icons.  If we don't include them we might have issues when using components from the library.  Should we add them back in?

Reply:
build size is important.

we should use icons from `Ionicons` unless the icon is not present there, in that case we can import just one another fonts/set of icons.

Reply:
I've raised a PR to add them back in https://github.com/GaloyMoney/galoy-mobile/pull/1184